tests/e2e.sh

```diff
@@ -118,6 +133,40 @@ assert_dir_exists payments-dev
 assert_dir_exists payments-prod
 grep -q allowPrivilegeEscalation podsecuritypolicy_psp.yaml
 
+helm_testcase "node_helm_template_command_line_args"
+node "${DIST}"/helm_template_run.js -i /dev/null -d name=my-chart -d chart_path=charts/bitnami/redis/ >out.yaml
+assert_contains_string out.yaml "my-chart"
+
+helm_testcase "node_helm_template_function_config_args"
+cat >fc.yaml <<EOF
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: my-config
+  annotations:
+    config.k8s.io/function: |
+      container:
+        image:  gcr.io/kpt-functions/helm-template
+    config.kubernetes.io/local-config: "true"
+data:
+  name: my-chart
+  chart_path: charts/bitnami/redis
+EOF
+node "${DIST}"/helm_template_run.js -i /dev/null -f fc.yaml >out.yaml
+assert_contains_string out.yaml "my-chart"
+
+helm_testcase "node_helm_template_undefined_args"
```

Comment:
why do we need non-dockerized tests? let's avoid duplication if possible.

Reply:
Converted to docker tests.